README.md

```diff
@@ -0,0 +1,18 @@
+Python Tic-Tac-Toe
+==================
+
+### Getting Started
+
+Run `python3 start.py` in the __PythonTTT__ folder.
```

Comment:
Is there any guarantee that the `__PythonTTT__`  folder will exist on my machine?

Reply:
I'll add a portion that explains the correct way to install. Git clone, cd to PythonTTT, Run python3 start.p